apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -305,29 +306,70 @@ class AktivitetspliktService(
         runBlocking { sendDtoTilStatistikk(sakId, brukerTokenInfo, behandlingId) }
     }
 
-    fun opprettUnntakForOpppgave(
+    fun upsertUnntakForOppgave(
         unntak: LagreAktivitetspliktUnntak,
         oppgaveId: UUID,
         sakId: SakId,
         brukerTokenInfo: BrukerTokenInfo,
-    ) {
+    ): AktivitetspliktVurdering {
         if (unntak.fom != null && unntak.tom != null && unntak.fom > unntak.tom) {
             throw TomErFoerFomException()
         }
+        val oppgave = oppgaveService.hentOppgave(oppgaveId)
```

Comment:
Burde denne også sjekket at oppgaveid.sakid == sakid param?
Evt kan det refaktoreres ut som en precondition for denne og `slettAktivitetsgradForOppgave`